### PR TITLE
277 - When coronavirus taxon present get priority breadcrumb

### PR DESCRIPTION
**Pull request by reggieb**
## What

- Updates `ContextualNavigation` to identify taxons that match particular content_ids
- Use that facility to identify a coronavirus taxon, and to use that to build a contextual bread crumb with the title and base url from the taxon.
- When more than one matching taxon is found, selected one is based on position in lookup list `GovukPublishingComponents::Presenters::ContentBreadcrumbsBasedOnPriority::PRIORITY_TAXONS`

## Why
The taxon superbreadcrumb will provide users with a link back up to coronavirus pages if they are tagged to part of the taxon tree.

### Page before change showing breadcrumbs
277-showing-original-breadcrumb

### Page after change showing new breadcrumb
277-showing-new-breadcrumb

https://trello.com/c/w3jmEyNT/277-add-super-breadcrumb-to-content-tagged-to-a-hub-page-or-hub-subtopic

### CODE REVIEW

File: lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+module GovukPublishingComponents
+  module Presenters
+    class ContentBreadcrumbsBasedOnPriority
+      # keys are labels, values are the content_ids for the matching taxons
+      # Where multiple matching taxons are present, the top most one is the highest priority
+      #   and the bottom one the lowest priority
+      PRIORITY_TAXONS = {
+        education_coronavirus: "272308f4-05c8-4d0d-abc7-b7c2e3ccd249",
+        business_coronavirus:  "65666cdf-b177-4d79-9687-b9c32805e450",
+        coronavirus:           "5b7b9532-a775-4bd2-a3aa-6ce380184b6c"
```
Comment (sihugh) [inline lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb:10]: I _think_ we can leave the link to the top level coronavirus taxon out now, given there's a banner to here on every page - covered in the slack comment from Sam:
> my only reservation with Simon's approach is that a superbreadcrumb linking to the homepage is duplicating the site-wide banner

**Comment by sihugh** [comment]
Approved pending product review

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
 * Add action link component ([PR #1497](https://github.com/alphagov/govuk_publishing_components/pull/1497))
 * Add `id` to checkboxes component ([PR #1503](https://github.com/alphagov/govuk_publishing_components/pull/1503))
+* Add priority breadcrumb ([PR #1501] (https://github.com/alphagov/govuk_publishing_components/pull/1501))
```
Comment (andysellick) [inline CHANGELOG.md:19]: This line needs to be in a new section above `21.48.0` as `Unreleased` i.e.

```
## Unreleased

* Add priority breadcrumb ([PR #1501] (https://github.com/alphagov/govuk_publishing_components/pull/1501))
```

**Comment by reggieb** [comment]
Product review caused delay in deployment while content was updated. That is not complete so merging.

**Pull request merged**